lib/economic/endpoint.rb

```diff
@@ -8,6 +8,10 @@ class Economic::Endpoint
   def_delegator "client.globals", :log_level, :log_level=
   def_delegator "client.globals", :log, :log=
 
+  def initialize(app_identifier = nil)
```

Comment:
A note in the README sounds good. I'd prefer some rdoc-style documentation directly in the code above the method definition if possible. See for example `#call` below. This way the docs will appear on for example http://www.rubydoc.info/gems/rconomic/Economic/Endpoint